monetio/sat/_mopitt_l3_mm.py

```diff
@@ -0,0 +1,147 @@
+""" MOPITT gridded data File reader
+    updated 2022-10 rrb
+         * DataSet instead of DataArray
+    created 2021-12 rrb
+"""
+import glob
+
+import pandas as pd
+import xarray as xr
+
+
+def getStartTime(filename):
+    """Method to read the time in MOPITT level 3 hdf files.
+
+    Parameters
+    ----------
+    filename : str
+        Path to the file.
+
+    Returns
+    -------
+    pandas.Timestamp or None or 0
+    """
+    import h5py
+
+    structure = "/HDFEOS/ADDITIONAL/FILE_ATTRIBUTES"
+    # print("READING FILE " + inFileName)
+    # fName = os.path.basename(filename)
+
+    try:
+        inFile = h5py.File(filename, "r")
+    except Exception:
+        print("ERROR: CANNOT OPEN " + filename)
+        return 0
+
+    grp = inFile[structure]
+    k = grp.attrs
+    startTimeBytes = k.get("StartTime", default=None)
+    startTime = pd.to_datetime(startTimeBytes[0], unit="s", origin="1993-01-01 00:00:00")
+    # print("******************", startTime)
+
+    try:
+        inFile.close()
+    except Exception:
+        print("ERROR CANNOT CLOSE " + filename)
+        return 0
+
+    return startTime
+
+
+def loadAndExtractGriddedHDF(filename, varname):
```

Comment:
```suggestion
def _open_single_dataset(filename, varname):
```
Seems like the intention is not to use this directly.